src/screens/Embed/Embed.js

```diff
@@ -61,11 +61,6 @@ function LocationEmbed() {
     return null;
   }
 
-  const primary = projections.primary;
-  if (!primary) {
-    return <span>'No data available for county.'</span>;
-  }
-
```

Comment:
Per my comment on the assert in model.ts, it's no longer possible to hit this case.  The API exposes data for all states/counties, even if there's no actual non-null data in it.